Doc.html/cards.html

```diff
@@ -0,0 +1,525 @@
+    <link rel="stylesheet" href="../Docs-css/Docs-main.css" />
+    <title>Cosmic UI | Cards</title>
+  </head>
+  <body>
+    <div class="outer-container">
+      <nav class="nav-bar">
+        <div class="nav-innerContainer">
+          <h2 class="nav-heading">Cosmic UI</h2>
+        </div>
+        <div class="nav-innerContainer">
+          <i class="fa fa-github nav-icon"></i>
+          <i class="fa fa-github nav-icon"></i>
+          <i class="fa fa-github nav-icon"></i>
+          <i class="fa fa-github nav-icon"></i>
+        </div>
+      </nav>
+
+      <div class="inner-container">
+        <aside class="sidebar customThemeBox">
+          <h2 class="drawer-title">Components</h2>
+
+          <ul>
+            <li class="list-item" >Alerts</li>
+            <li class="list-item" >Avatar</li>
+            <li class="list-item" >Badge</li>
+            <li class="list-item" >Button</li>
+            <li class="list-item" >Cards</li>
+            <li class="list-item" >Input</li>
+            <li class="list-item" >Image</li>
+          </ul>
+        </aside>
+
+        <main class="main-container customThemeBox">
+          <div id="cards">
+            <h1 class="title">Cards</h1>
+            <p class="subtitle">
+              Cards are used to display content as well as it gives user an
+              option to take action on that content, like e-commerce product
+              card
+            </p>
+
+            <h2 class="small-title">Vertical Card</h2>
+            <p class="para">
+              Vertical card with image, content and call to action buttons,there
+              are 2 variants blog card and e-commerce card, to use copy the
+              below html code.
+            </p>
+
+            <div class="container">
+              <div class="card-basic">
+                <div class="">
+                  <img
+                    class="card-img"
+                    src="https://media.macphun.com/img/uploads/customer/blog/347/15361508215b8fcd25b13f46.53184133.jpg?q=85&w=1680"
+                  />
+
+                  <div class="">
+                    <h3 class="card-heading">Our Changing Planet</h3>
+                    <small class="card-subHeading">by Kurt Wagner</small>
+                  </div>
+
+                  <p class="card-description">
+                    Visit ten places on our planet that are undergoing the
+                    biggest changes today.
+                  </p>
+                </div>
+
+                <div class="card-footer">
+                  <button class="card-btn">READ</button>
+                  <button class="card-btn">BOOKMARK</button>
+                  <img
+                    class="heart-icon"
+                    src="https://img.icons8.com/fluency-systems-regular/50/000000/like--v1.png"
+                  />
+                  <img src="../Components/icons/share.png" alt="share icon" />
+                  <img src="../Components/icons/more.png" alt="more icon" />
```

Comment:
instead of having png, you can use resources like https://boxicons.com/ for icons.

Reply:
Thanks for suggesting i will use this.